config/service.go

```diff
@@ -64,6 +71,23 @@ func (srvTL *ServiceTelemetryLogs) validate() error {
 	return nil
 }
 
+// ServiceTelemetryMetrics exposes the common Telemetry configuration for one component.
+type ServiceTelemetryMetrics struct {
+	// Level is the level of telemetry metrics, the possible values are:
+	//  - "none" indicates that no telemetry data should be collected;
+	//  - "basic" is the recommended and covers the basics of the service telemetry.
+	//  - "normal" adds some other indicators on top of basic.
+	//  - "detailed" adds dimensions and views to the previous levels.
+	Level configtelemetry.Level `mapstructure:"level"`
+
+	// Address is the [address]:port that metrics exposition should be bound to.
+	Address string `mapstructure:"address"`
```

Comment:
Address, or "metrics-addr", was the terminology used in the CLI flags this is derived from.  I could see it either way.

Reply:
I think we can keep it as is. Most likely we will have to change this and it will be a breaking change no matter what, so as a temporarily solution this is fine.